### DISCUSSION TITLE
NOVA ITQB template

### DISCUSSION THREAD

**Discussion by luisalisboa-mcr** [joaomlourenco/novathesis#460] [Q&A] [answered]
Hii, 

I'm trying to set up my template for my master thesis a NOVA but the cover page is "stuck" in PhD - so it seems I'm submitting for a doctorate and not a master's. How can I change it? I'm using a ITQB cover. 
Thanks you

**Comment by joaomlourenco** [comment]
Did you change the `doctype` option in `Config/1_novathesis.tex`??

**Reply by luisalisboa-mcr** [reply]
I did yes! however now I can't compile the file anymore. 
![imagem](https://github.com/joaomlourenco/novathesis/assets/107812281/91d05811-4bf4-467e-ab80-e7669392e23e)

**Comment by joaomlourenco** [comment]
Just tried and it works.  Please start over again changing only what is visible in the image below.
<img width="1489" alt="Screenshot 2023-09-14 at 11 43 43" src="https://github.com/joaomlourenco/novathesis/assets/2064643/32432553-4dd0-41d5-a958-c489d31e616c">

**Reply by luisalisboa-mcr** [reply]
Thank you I'll start over :)

**Comment by joaomlourenco** [comment, marked as answer]
My mistake, you wrote "nova itqb" and I read "nova ims".
But it works for nova itqb.
![image](https://github.com/joaomlourenco/novathesis/assets/2064643/eb93aac0-ecbd-4429-8f04-4b77cc6bd8da)
